application-diagram-ui/src/main/resources/Diagram/DiagramSheet.xml

```diff
@@ -634,27 +634,45 @@ define('diagram-graph-xml-filter', ['jquery', 'diagram-config', 'diagram-link-ha
   };
 
   // Replace custom internal links with absolute URLs (but keep some metadata to indicate the entity reference).
+  // Consider both the cases when the node is UserObject or mxCell.
   var onExportLink = function(node) {
-    var link = $(node).attr('link');
+    var tagName = node.tagName.toLowerCase();
+    var value = $(node).attr('value');
+    if (value) {
+      var href = $('&lt;div&gt;&lt;/div&gt;').append(value).find('a[href]').attr('href');
```

Comment:
jQuery functions are null-safe in general so I'm not sure you need to check if there is a value. Also, I would use ``.html()`` instead of ``.append()`` but both work fine.